omnibus/files/private-chef-ctl-commands/cleanup_bifrost.rb

```diff
@@ -0,0 +1,231 @@
+require 'pg'
+
+CREATE_SQL =<<SQL
+BEGIN;
+CREATE UNLOGGED TABLE IF NOT EXISTS cleanup_tracking_auth_actors(
+    authz_id  CHAR(32)
+);
+
+CREATE OR REPLACE RULE cleanup_auth_actor_creation_tracking AS ON INSERT TO auth_actor DO INSERT INTO cleanup_tracking_auth_actors VALUES (NEW.authz_id);
+COMMIT;
+SQL
+
+CREATE_SQL2 =<<SQL
+CREATE UNLOGGED TABLE IF NOT EXISTS cleanup_known_auth_actors(
```

Comment:
It may be cleaner to re-use the cleanup_tracking_auth_actors table for tracking known actors as well.  That will also save a UNION in the delete query.